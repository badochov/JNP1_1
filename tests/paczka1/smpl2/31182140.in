9oVnA7mISWz A343 14,5
A2IUytpV A343 25,2
X4IeUrDcrI0 A724 74,9
A2IUytpV A343 42,8
X4IeUrDcrI0 A724 15,7
UtQOSKlNCo S106 91,6
UtQOSKlNCo A343 15,8
UtQOSKlNCo A724 13,6
UtQOSKlNCo A343 87,4
9oVnA7mISWz A343 44,7
S119 S106 59,0

S119 S106 58,4
A2IUytpV S119 65,0


S119 S106 175,8
?FINxWB
sxG4voq A724 13,6
sxG4voq S119 55,5
UtQOSKlNCo A343 72,7
A2IUytpV S119 14,0
A2IUytpV A724 21,0
9oVnA7mISWz S106 28,5
9oVnA7mISWz S106 36,5
S119 A343 27,5

sxG4voq S119 271,4
A2IUytpV A724 68,7
S119 S106 91,3
UtQOSKlNCo A343 205,7
A2IUytpV S106 93,1
S119 S106 140,6
S119 S106 1,3
A2IUytpV S106 -1,1
X4IeUrDcrI0 S106 7,0
A2IUytpV A724 15,7
S119 S106 4,9
?S119
A2IUytpV A724 17,0