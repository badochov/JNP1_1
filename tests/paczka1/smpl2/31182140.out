S119 S 53,5
S119 266,9
